### PR TITLE
[ADD] search function supporting case insensitive login

### PR DESCRIPTION
**Pull request by gfcapalbo**
Case insensitive module allows to create/write lowercase logins in DB.
After that the input is sanitized to be allways lowercase.

This extension adds search, so that it is not necessary anymore to lowercase all inputs, but the input sanitization is built-into the search function if needed.

Very useful for projects for with several login methods/interfaces.

### CODE REVIEW

**Review by hbrunn — APPROVED** [review]

File: auth_user_case_insensitive/models/res_users.py [hunk 1]
```diff
@@ -32,3 +32,24 @@ def write(self, vals):
         if vals.get('login'):
             vals['login'] = vals['login'].lower()
         return super(ResUsers, self).write(vals)
+
+    @api.model
+    def search(self, args, offset=0, limit=None, order=None, count=False):
+        """
```
Comment (NL66278) [inline auth_user_case_insensitive/models/res_users.py]: Format this according to PEP rules for doc comments (using full sentences, If possible one line, etc.).

Comment (gfcapalbo) [reply auth_user_case_insensitive/models/res_users.py]: now compliant with https://www.python.org/dev/peps/pep-0257/

**Review by NL66278 — APPROVED** [review]
:+1:

**Pull request merged**